packages/frontend/src/pages/Smmart/Smmart.tsx

```diff
@@ -1,69 +1,69 @@
 import { GetServerSideProps } from 'next';
-import { MantineProvider, Container, Text, Grid } from '@mantine/core';
+import { MantineProvider, Container, Text, Button, Grid, Card, Image, Flex } from '@mantine/core';
+import {
+  NavPageLayout,
+  NavPageLayoutProps,
+  getNavPageLayoutPropsFromConfig,
+} from '@gen3/frontend';
```

Comment:
do you want to keep only the changes from the original page since this is ours? It might be good to put this in the sampleCommons in the future so it doesn't get touched? Idk why it's being edited